python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
and really, I don't like the idea of spawning a subshell to do something that can be done fairly straightforwardly in python itself. Slurping the file, running `sorted` and writing it out again. Using the native language is also more portable than relying on external applications.